src/components/CharacterItem/CharacterItem.tsx

```diff
@@ -77,8 +76,8 @@ const CharacterItem = ({ character }: CharacterItemProps) => {
           p={6}
           textAlign={'center'}
           border={'1px'}
-          borderColor='black'
-          boxShadow={useColorModeValue('6px 6px 0 black', '6px 6px 0 cyan')}
+          borderColor='black.900'
+          boxShadow={useColorModeValue('6px 6px 0 #221F20', '6px 6px 0 cyan')}
```

Comment:
I saw you added this color to a theme. Is there any particular reason why you didn't use it here?

Reply:
This is just because 'black.900' doesn't work with this syntax 😅

Reply:
Ah kk